src/tests/ftest/util/dfuse_utils.py

```diff
@@ -264,6 +264,11 @@ def run(self, check=True):
             raise CommandFailure(
                 "Dfuse missing environment variables for D_LOG_FILE")
 
+        if 'D_LOG_MASK' not in self.env:
+            self.env['D_LOG_MASK'] = 'INFO'
+        # self.env['DD_MASK'] = 'all'
+        # self.env['DD_SUBSYS'] = 'all'
```

Comment:
[MINOR] May be leftovers. Should be removed.

Reply:
I discussed the Dalton, I often find I need to enable these for dfuse tests and struggle to know where to put the code, these were intentional and intended as a reminder to me, although it's not a very good solution.  Will remove.